code/src/Core/PostActions/Catalog/NewItemGeneration/CreateSyncSummaryPostAction.cs

```diff
@@ -141,19 +134,14 @@ public override void Execute()
             GenContext.Current.MergeFilesFromProject.Clear();
         }
 
-        private static string GetLinkToProjectFile(string fileName, string filePath)
-        {
-            return $"* [{fileName}]({FormatFilePath(filePath)})";
-        }
-
         private static string GetCompareLink(string fileName, string filePath)
         {
-            return $"* {StringRes.SyncSummaryTempGenerationFile}: [{fileName}]({fileName}), {StringRes.SyncSummaryProjectFile}: [{fileName}]({FormatFilePath(filePath)})";
+            return $"* {StringRes.SyncSummaryTempGenerationFile}: [{fileName}]({fileName}), {StringRes.SyncSummaryProjectFile}: {GetMarkDownLinkToLocalFile(filePath)}";
         }
 
-        private static string FormatFilePath(string filePath)
+        private static string GetMarkDownLinkToLocalFile(string fileName)
         {
-            return $"about:/{filePath.Replace(" ", "%20").Replace(@"\", "/")}";
+            return $"* [{fileName}]({fileName})";
```

Comment:
I agree. If the changes are merged in the project, we should open the project file, not the external copy which would lead to confusion.  

@crutkas What do you think?